Pipelines/YmlTemplates/deploy.yml

```diff
@@ -0,0 +1,86 @@
+          value: ""
+        ${{ if not( eq(parameters.pipelineDependency, 'current')) }}:
+          value: ${{ parameters.pipelineDependency }}
+    jobs:
+      - deployment: SDKNugetPublish
+        displayName: Publishing on ${{ parameters.nuGetFeedType }} Feed
+        environment: ${{ format('NfieldSDK{0}Feed', parameters.nuGetFeedType) }}
+        continueOnError: false
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                # Download the 'for-deploy' artifacts
+                - download: ${{ parameters.pipelineDependency }}
+                  artifact: for-deploy
+                  patterns: |
+                    **/for-deploy/**
+                    **/Pipelines/**
+                # Publish NuGet package
+                - task: NuGetCommand@2
+                  displayName: 'NuGet push'
+                  continueOnError: True
+                  inputs:
+                    command: push
+                    packagesToPush: '$(Pipeline.Workspace)/$(baseRoute)/for-deploy/*.nupkg'
+                    nuGetFeedType: ${{ parameters.nuGetFeedType }}
+                    publishFeedCredentials: 'NuGet (Publish)' # Used only for external feed
+                    publishVstsFeed: 'nfield' # Used only for internal feed
+                  condition: succeeded()
+
+                - task: PowerShell@2
+                  displayName: 'Gather Release Info'
+                  inputs:
+                    targetType: 'inline'
+                    script: |
+                      $GitReleaseInfo = Get-Content -Path $(Pipeline.Workspace)/$(baseRoute)/for-deploy/NugetReleaseInfo.txt
+                      Write-Host $GitReleaseInfo
+                      $InfoFields = $GitReleaseInfo.Split(":")
+                      $BranchName = $InfoFields[0]
+                      $ReleaseName = $InfoFields[1]
+                      $CommitHash = $InfoFields[2]
+                      $VersionName = $InfoFields[3]
+                      Write-Host BranchName: $BranchName
+                      Write-Host ReleaseName: $ReleaseName
+                      Write-Host CommitHash: $CommitHash
+                      Write-Host VersionName: $VersionName
+                      Write-Host "##vso[task.setvariable variable=BranchName]$BranchName"
+                      Write-Host "##vso[task.setvariable variable=ReleaseName]$ReleaseName"
+                      Write-Host "##vso[task.setvariable variable=CommitHash]$CommitHash"
+                      Write-Host "##vso[task.setvariable variable=VersionName]$VersionName"
+                      
+                - task: GitHubRelease@1
+                  displayName: 'Create GitHub Release'
+                  condition: ${{ eq(parameters.nuGetFeedType, 'external') }}
+                  inputs:
+                    gitHubConnection: 'github.com_AlbertoBelver' # OAuth Service Conection https://dev.azure.com/niposoftware/Nfield/_settings/adminservices
```

Comment:
Shouldn't we have a proper github connection for that? And not a personal one?

Reply:
We don't have any shared github account to do it. I want to discuss it in the SoS meeting. By the moment I leave it as it is.
Is also happening with Github PATs

Reply:
https://github.com/NIPOSoftwareBV/Nfield-Documentation/blob/master/meeting-notes/Retros/SoS-discussion-points.md